ApollosProject/apollos-prototype
# Adds Pagination w/ First and Limit support to the getLikedContent

## DESCRIPTION

Adds Pagination w/ First and Limit support to the getLikedContent. This query was originally built without this feature b/c it was deemed too difficult to implement with the interactions feature. 

Now that we are using the `Following` feature, we figured we could add pagination, which has been done in this PR. This change will requires updates to the UI of any implementor who updates the server packages. 

### What does this PR do, or why is it needed?

Changes the getLikedContent query to support pagination, changing UI to handle the new, paginated query.

### What design trade-offs/decisions were made?

Did not support "fetch next page" at this time, seeing as no other implementations of `feedView` have implemented. 

### How do I test this PR?

1. Open the app.
2. Sign in.
3. Like some content.
4. Visit your `Connect` page
5. Ensure items show up in your liked content items list
6. Unlike an item
7. Ensure an item is removed from the liked content items list.

### What automated tests did you write?

## TODO:

- [ ] Closes #330

## REVIEW:

**Manual QA**


**Code Review: Questions to consider**


> The purpose of PR Review is to _improve the quality of the software._
